lib/appsignal/rack/generic_instrumentation.rb

```diff
@@ -5,42 +5,15 @@
 module Appsignal
   # @api private
   module Rack
-    class GenericInstrumentation
+    class GenericInstrumentation < AbstractMiddleware
       def initialize(app, options = {})
-        Appsignal.internal_logger.debug "Initializing Appsignal::Rack::GenericInstrumentation"
-        @app = app
-        @options = options
+        options[:instrument_span_name] ||= "process_action.generic"
```

Comment:
Perhaps we could use `.rack` as the category group, which [already has a defined color](https://github.com/appsignal/appsignal-server/blob/55059feff5f8ff579372ac201549c67d91688a02/app/frontend/javascripts/constants/EventGroups.js#L26-L28).

Reply:
I want to go through all these event names some time later, because they report `process_action` which makes no sense for a rack middleware. The `generic` group isn't a good name either, we can change it to rack, but I'd rather do that later.